src/d3a/setup/json_file.py

```diff
@@ -25,7 +25,7 @@ def get_setup(config):
         with open(setup_path, 'r') as area_file:
             area_str = area_file.read().replace('\n', '')
         recovered_area = area_from_string(area_str)
-        recovered_area.config = config
+        recovered_area._config = config
```

Comment:
This got through because nobody used this ever, right?

Reply:
Exactly, this hasn't been used for a while now. I find it convenient for replicating errors in the production environment though, that's why I thought to reintroduce it.